solarwinds_apm/trace/response_time_processor.py

```diff
@@ -53,21 +61,15 @@ def __init__(
         )
 
     def is_span_http(self, span: "ReadableSpan") -> bool:
-        """This span from inbound HTTP request if from a SERVER by some http.method"""
+        """This span from inbound HTTP request if from a SERVER by some http.request.method or http.method"""
         return bool(
             span.kind == SpanKind.SERVER
-            and span.attributes.get(self._HTTP_METHOD, None)
+            and (
+                span.attributes.get(self._HTTP_REQUEST_METHOD, None)
+                or span.attributes.get(self._HTTP_METHOD, None)
+            )
         )
 
-    def get_http_status_code(self, span: "ReadableSpan") -> int:
```

Comment:
Helper somewhat replaced by `enhance_meter_attrs_with_http_span_attrs`, so removed and added tests accordingly